README.md

```diff
@@ -14,243 +14,222 @@
 
 #### Services
 For each instance of the component the following services will be available:
+
 | Service | Description|
 | --- | --- |
-|`smart_irrigation.[instance]_calculate_daily_adjusted_run_time`|Triggers the calculation of daily adjusted run time. Use only if you disabled automatic refresh in the options.|
-|`smart_irrigation.[instance]_calculate_hourly_adjusted_run_time`|Triggers the calculation of hourly adjusted run time. Use for debugging only.|
-|`smart_irrigation.[instance]_enable_force_mode`|Enables force mode.|
-|`smart_irrigation.[instance]_disable_force_mode`|Disables force mode.|
-|`smart_irrigation.]instance]_reset_bucket`|Resets the bucket to 0. Needs to be called after done irrigating (see below).|
-|`smart_irrigation.[instance]_set_bucket`|Sets the bucket to the provided `value`. Use for debugging only.|
-
-#### Events
-The component uses a number of events internally that you do not need to pay attention to unless you need to debug things. The exception is the `_start` event.
+|`Smart Irrigation: calculate_all_zones`|Triggers the calculation of all zones. Use only if you disabled automatic refresh in the options.|
+|`Smart Irrigation: calculate_zone`|Triggers the calculation of one specific zone.|
+|`Smart Irrigation: reset_all_buckets`|Resets all buckets.|
+|`Smart Irrigation: reset_bucket`|Resets one specific bucket.|
+|`Smart Irrigation: set_all_buckets`|Sets all buckets to a specific `value`.|
+|`Smart Irrigation: set_bucket`|Sets a specific bucket to to a specific `value`.|
+|`Smart Irrigation: update_all_zones`|Updates all zones with weather data|
+
+#### Start Event
+
 | Event | Description|
 | --- | --- |
-|`[instance]_start`|Fires depending on `daily_adjusted_run_time` value and sunrise. You can listen to this event to optimize the moment you irrigate so your irrigation starts just before sunrise and is completed at sunrise. See below for examples on how to use this.|
-|`[instance]_bucketUpd`|Fired when the bucket is calculated. Happens at automatic refresh time or as a result of the `reset_bucket`, `set_bucket` or `calculate_daily_adjusted_run_time` service.|
-|`[instance]_forceModeTog`|Fired when the force mode is disabled or enabled. Result of calling `enable_force_mode` or `disable_force_mode`|
-|`[instance]_hourlyUpd`|Fired when the hourly adjusted run time is calculated. Happens approximately every hour and when `calculate_hourly_adjusted_run_time` service is called.|
+|`[instance]_start`|Fires depending on `sensor.[zone_name]` value and sunrise. You can listen to this event to optimize the moment you irrigate so your irrigation starts just before sunrise and is completed at sunrise. See below for examples on how to use this.|
 
 #### Entities
-#### `sensor.smart_irrigation_base_schedule_index`
+#### `sensor.[zone_name]`
 The number of seconds the irrigation system needs to run assuming maximum evapotranspiration and no rain / snow. This value and the attributes are static for your configuration.
```

Comment:
delete - this is about BSI which is not there any more.

Reply:
done
With v2.0 README is getting shorter and shorter. Good job! :-)

Reply:
well, our wiki will grow and grow :)